modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/scheduler.js

```diff
@@ -1565,6 +1565,10 @@ AUI.add(
 					_onClickAddEvent: function(event) {
 						var instance = this;
 
+						if(Liferay.Session.get('sessionState') === "expired") {
+							return;
+						}
+						
```

Comment:
Hey @linolaoj! You just forgot to run gradlew formatSource: there were some spurious whitespaces here.

Besides that, the fix is correct. Thanks!